tests/ui/let-else/accidental-if.stderr

```diff
@@ -0,0 +1,19 @@
+error: this `if` expression is missing a block after the condition
+  --> $DIR/accidental-if.rs:3:5
+   |
+LL |     if let Some(y) = x else {
+   |     ^^
+   |
+help: add a block here
+  --> $DIR/accidental-if.rs:3:23
+   |
+LL |     if let Some(y) = x else {
+   |                       ^
+help: There may be an extra `if` before the `let...else`
```

Comment:
This can be misinterpreted in the way that the statement is saying that you are allowed to put `if` in front of `let...else`, instead of what the statement *intends* to say, of the author potentially having mistakenly put an `if` in front of `let...` else.

I wonder if there are better ways to write this suggestion that can't be misinterpreted.

Reply:
Thanks for the feedback! How about:

```
help: Remove the `if` to construct a `let...else` statement
```

I think that's a little more direct while still sounding neutral.

Reply:
@edward-shen that is perfect!